src/basic/cgroup-util.h

```diff
@@ -331,6 +331,16 @@ typedef enum ManagedOOMPreference {
 const char* managed_oom_preference_to_string(ManagedOOMPreference a) _const_;
 ManagedOOMPreference managed_oom_preference_from_string(const char *s) _pure_;
 
+typedef enum IOCostCtrl {
+        IO_COST_CTRL_AUTO,
+        IO_COST_CTRL_USER,
+        _IO_COST_CTRL_MAX,
+        _IO_COST_CTRL_INVALID,
```

Comment:
We set the invalid value of an enum equal to `-EINVAL` almost everywhere so we should do the same here as well unless there's a good reason not to.